internal/conf/jwk.go

```diff
@@ -0,0 +1,192 @@
+package conf
+
+import (
+	"crypto/ecdh"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+)
+
+type JwtKeysDecoder map[string]JwkInfo
+
+type JwkInfo struct {
+	PublicKey  jwk.Key `json:"public_key"`
+	PrivateKey jwk.Key `json:"private_key"`
+}
+
+// Decode implements the Decoder interface
+// which transforms the keys stored as der binary strings into jwks
+func (j *JwtKeysDecoder) Decode(value string) error {
+	data := make([]map[string]interface{}, 0)
+	if err := json.Unmarshal([]byte(value), &data); err != nil {
+		return err
+	}
+
+	config := JwtKeysDecoder{}
+	for _, key := range data {
+		bytes, err := json.Marshal(key)
+		if err != nil {
+			return err
+		}
```

Comment:
Why are you unmarshalling then marshalling again? You can just use something like:

`var data []json.RawMessage`, then unmarshal the string in the data, and just do `jwk.ParseKey(data[i])`?